src/styles/blocks/cards-container.scss

```
@@ -0,0 +1,24 @@
+.cards-container{
+  --columns: 1;
+
+  display: grid;
+  grid-template-columns: repeat(var(--columns), 1fr);
```

just use `card-width` here
```suggestion
  grid-template-columns: repeat(var(--columns), ...);
```